python/tests/integration/test_client_server.py

```diff
@@ -111,27 +112,63 @@ def test_tchannel_call_request(tcp_server, call_response):
     hostport = 'localhost:%d' % (tcp_server.port)
 
     response = yield tchannel.request(hostport).send(endpoint, None, None)
+    assert response.args[0] == call_response.args[0]
+    assert response.args[2] == call_response.args[2]
 
-    assert response.arg_1 == call_response.arg_1
-    assert response.arg_3 == call_response.arg_3
+
+@pytest.fixture
+def load_data_from_file():
```

Comment:
Why does this exist? You're not using it anywhere.

Also, this is a fixture that returns `None` so the caller wouldn't work anyway.